### ISSUE TITLE
passes closes itself when deleting 2nd pass

### ISSUE THREAD

**Issue by ronald** [pablo-s/passes#49]
I'm developing a pass and testing using passes.
Every time I delete a second pass the app quits.
Using version 0.9 installed via flatpak.

Is there a way to get more information / debugging logs?

**Comment by pablo-s** [comment]
Hello @ronald ! Thanks for reporting an issue.

When you say "the second pass"... how many of them you have? Is it the second of 2? Or you have more?

Currently Passes is not producing log files, but please, try to run the application from the command line and check the possible errors:

```
flatpak run me.sanchezrodriguez.passes
```

**Comment by pablo-s** [comment]
I have just tried to reproduce the error with the last development changes and the application seems to work as expected.

If after you run it on the command line we do not find information that allows us to properly identify and fix this possible bug, I will try to release a new version. Hopefully this will improve the situation for you.

**Comment by ronald** [comment]
Hello and thanks for your feedback and your work.

I've created a lot of passed with different IDs to visually compare them. 
Yesterday I stumbled about the bug multiple times and I already tried to get output using 
`flatpak run me.sanchezrodriguez.passes`
but I think I got nothing.

Today I tried to reproduce it with the current version of my passes but passes is working as expected.

I'll close the issue for now (and will reopen if I'm able to reproduce again).
